lib/swiftype-app-search/request.rb

```diff
@@ -86,6 +87,29 @@ def debug?
       @debug ||= (ENV['AS_DEBUG'] == 'true')
     end
 
+    def serialize_json(object)
+      JSON.generate(clean_json(object))
+    end
+
+    def clean_json(object)
+      case object
+      when Hash
+        object.transform_values { |value| clean_json(value) }
```

Comment:
When was `transform_values` introduced in Ruby? We may want to use something that supports older versions

Reply:
It was added in Ruby 2.4. I changed it to use `inject`